### PR TITLE
weapon icon esp

### PR DESCRIPTION
**Pull request by VegifZyx1336**
the spacing not perfect but it ight <-- Made by toggle(Toggle17) he couldnt make pull request on it so he asked me to do it

### CODE REVIEW

File: Osiris/GameData.cpp [hunk 1]
```diff
@@ -538,10 +538,112 @@ void PlayerData::update(Entity* entity) noexcept
...
+            case WeaponId::NomadKnife:    return   "1   ";
+            case WeaponId::SkeletonKnife: return   "1   ";
+            case WeaponId::SurvivalKnife: return   "1   ";
+            case WeaponId::Ursus:         return   "1   ";
+            case WeaponId::Stiletto:      return   "1   ";
+            case WeaponId::Talon:         return   "1   ";
+            case WeaponId::Paracord:      return   "1   ";
+            case WeaponId::Navaja:        return   "1   ";
+
+            case WeaponId::Taser:         return "h";
```
Comment (Xsintashi) [inline Osiris/GameData.cpp:632]: 💀

Comment (VegifZyx1336) [reply Osiris/GameData.cpp:632]: gang

**Comment by St0rmmx** [comment]
mid

**Comment by Wolfyyta** [comment]
what the fuck am I watching looks like someone just found a random paste and ported the shit from it here

**Comment by Xsintashi** [comment]
> what the fuck am I watching looks like someone just found a random paste and ported the shit from it here

cuz its pasted

**Comment by Liam5750** [comment]
this is disgusting. I fixed the for loop and centering and its decent.

**Pull request closed without merging**